skimage/restoration/inpaint.py

```diff
@@ -40,7 +40,36 @@ def _get_neigh_coef(shape, center, coef_cache={}):
 
 def _inpaint_biharmonic_single_region(image, mask, out, neigh_coef_full,
                                       coef_vals, raveled_offsets, limits):
-    # Initialize sparse matrices
+    """Solve a (sparse) linear system corresponding to biharmonic inpainting.
+
+    This function creates a linear system of the form:
+
+    ``A @ u = b``
+
+    where ``A`` is a sparse matrix, ``b`` is a vector enforcing smoothness and
+    boundary constraints and ``u`` is the vector of inpainted values to be
+    (uniquely) determined by solving the linear system.
+
```

Comment:
Thanks! These explanations really help. Optionally, one additional sentence like:
"The linear system is obtained from a biharmonic equation Mx = 0 where x is the vector of all image pixels, and decomposing M and x into blocks corresponding respectively to known values and pixels to be inpainted."

Reply:
I got a bit confused by this suggested addition. Is `M` here supposed to be the same as `A` in the docstring? I think the Mx=0 constraint only applies to the inpainted pixels, not "all image pixels"?